docs/architecture.md

```diff
@@ -29,73 +29,100 @@ have at least 3 nodes. In a basic setup with this amount of nodes,
+
+For storage, the Operator leverages Persistent Volumes (PVs) and PersistentVolumeClaims (PVCs) to ensure your data is durable. A *PersistentVolumeClaim* (PVC) is used to implement
+the automatic storage provisioning to pods. The Container Storage Interface (CSI) allows the storage to be re-mounted on a different node if a failure occurs.
 
 ## Replication types and proxy solutions
 
 The process in which data from a *primary* MySQL instance is copied and sent to
-other MySQL instances of the database cluster (co-called *replicas*) is known as
+other MySQL instances of the database cluster (so-called *replicas*) is known as
 *MySQL replication*.
 
 The Operator [provides you a choice](operator.md#mysqlclustertype) between two
-replication types: 
+replication types:
 
-* **Asynchronous replication** means that write is completed on the
+* **Asynchronous replication** means that a write is completed on the
     primary and is not influenced by the replicas themselves. After completion
     of its changes, the primary populates the binary log with the data
     modification or the actual statement. Then the replica executes each change
     on its own database and obtains consistent set of data.
 
-* **Group Replication** means that read-write transaction can be executed on any
+* **Group Replication** means that a read-write transaction can be executed on any
     instance, and when it happens, MySQL tries to get consensus with the other
-    instances before returning it completed back to the client. 
+    instances before returning it completed back to the client.
 
 Both replication types have their pros and cons.
 
 *Asynchronous replication* is faster. Also, you can distribute read and write
-requests of your application to different instances. From the other hand, it can
+requests of your application to different instances. On the other hand, it can
 be less reliable in terms of consistency: replicas may lag behind the primary
 instance, impacting any applications that depend on real-time data. Also, some
 transaction committed on the primary instance may not be available on replicas
 if the primary fails. Finally, asynchronous replication doesn't allow you
-scaling write requests horizontally, leaving  vertical scaling (primary
+scaling write requests horizontally, leaving vertical scaling (primary
 instance RAM and CPU increase) as the only available option.
 
-*Group replication* allows horizontal scaling of writes, but is not so fast,
-because writes require consensus (so-called "certification").
-Also, concurrent writes of the same row on different instances are
-[treated as data conflicts :octicons-link-external-16:](https://dev.mysql.com/doc/refman/8.0/en/group-replication-summary.html):
-the transaction that was ordered first commits on all servers, and the
-transaction ordered second aborts. Finally, group replication has some
+*Group replication* allows horizontal scaling of reads, but is slower. Group replication has some
```

Comment:
*Group replication* allows horizontal scaling of reads, preventing stale reads when set with high consistency level, but it is slower.

Reply:
@nastena1606 my fault the exact term is Transaction Consistency so the phrase should be:
Group replication allows horizontal scaling of reads, preventing stale reads when set with high Transaction consistency level, but it is slower.